### PR TITLE
Luda editor with server

### PR DESCRIPTION
**Pull request by namse**
# What I did
- I Added luda editor server
- Added CI code to run deploy to aws
- Removed all github based code

### CODE REVIEW

File: luda-editor/client/src/app/login.rs [hunk 1]
```diff
@@ -1,14 +1,11 @@
 use super::App;
-use crate::storage::Storage;
-use editor_core::{github_client::GithubClient, storage::GithubStorage};
 use namui::prelude::*;
-use std::sync::Arc;
 
-const CLIENT_ID: &str = "abd04a6aeba3e99f5b4b";
-const CLIENT_SECRET: &str = "501a915ca627e24d2088cf01416fe836db470dba";
+const DEV_CLIENT_ID: &str = "abd04a6aeba3e99f5b4b";
+const CLIENT_ID: Option<&str> = option_env!("GITHUB_CLIENT_ID");
```
Comment (namse) [inline luda-editor/client/src/app/login.rs:5]: FYI: option_env can get environment variable on compile time.

File: luda-editor/client/src/app/login.rs [hunk 2]
```diff
@@ -51,35 +39,61 @@ impl App {
...
+                }
+            },
         }
     });
 }
+
+async fn request_github_auth_code() -> Result<String, Box<dyn std::error::Error>> {
+    let client_id = CLIENT_ID.unwrap_or(DEV_CLIENT_ID);
+    let redirect_uri = web_sys::window().unwrap().location().href().unwrap();
+    let url = format!("https://github.com/login/oauth/authorize?client_id={client_id}&redirect_uri=https://sslwiheugl5ojmqlecerzhng740cekqc.lambda-url.ap-northeast-2.on.aws/{redirect_uri}");
```
Comment (namse) [inline luda-editor/client/src/app/login.rs:59]: FYI: https://sslwiheugl5ojmqlecerzhng740cekqc.lambda-url.ap-northeast-2.on.aws/{redirect_uri} is our redirector url. It's on our company AWS account's Lambda.

Comment (namse) [reply luda-editor/client/src/app/login.rs:59]: This is for multiple oauth callback. github only support one callback, so I put redirector for local and test server.

File: luda-editor/client/src/pages/project_list_page/mod.rs [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+                                item_wh: Wh::new(wh.width, 40.px()),
+                                items: self.project_list.iter(),
+                                item_render: |wh, project| self.render_project_cell(wh, project),
+                            })
+                        }),
+                    ]),
+                ),
+                table::ratio(1.0, |_wh| RenderingTree::Empty),
+            ])(props.wh),
+            // self.context_menu
```
Comment (namse) [inline luda-editor/client/src/pages/project_list_page/mod.rs:102]: It's todo

File: luda-editor/client/src/pages/sequence_edit_page/loaded/update/mod.rs [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+                                }
+                            });
+                    self.editor_history_system
+                        .mutate_cut(selected_cut_id.unwrap(), |cut| {
+                            cut.line = updated.text.clone();
+                        });
+                }
+            }
+        }
+        //         else if let Some(event) = event.downcast_ref::<crate::storage::Event>() {
```
Comment (namse) [inline luda-editor/client/src/pages/sequence_edit_page/loaded/update/mod.rs:56]: It's TODO

File: luda-editor/core/src/lib.rs [hunk 5]
```diff
@@ -1,5 +1,5 @@
-pub mod github_client;
-pub mod storage;
+// pub mod github_client;
+// pub mod storage;
```
Comment (namse) [inline luda-editor/core/src/lib.rs]: will remove next time

File: luda-editor/server/lambda-web/CHANGELOG.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
+# Changelog
```
Comment (namse) [inline luda-editor/server/lambda-web/CHANGELOG.md:1]: Don't review lambda-web. it's fork of https://github.com/hanabu/lambda-web. There were a little trouble with lambda-web so I cloned and edit little code. Pull requested https://github.com/hanabu/lambda-web/pull/13

File: namui-cli/src/services/wasm_watch_build_service.rs [hunk 7]
```diff
@@ -123,6 +123,11 @@ impl WasmWatchBuildService {
         let runtime_target_dir = project_root_path.join("target/namui");
         let rust_build_service = RustBuildService::new();
 
+        runtime_project::wasm::generate_runtime_project(GenerateRuntimeProjectArgs {
```
Comment (namse) [inline namui-cli/src/services/wasm_watch_build_service.rs:126]: It's https://github.com/NamseEnt/namseent/pull/319

File: luda-editor/server/update-lambda.sh [hunk 8]
```diff
@@ -0,0 +1,2 @@
+aws lambda update-function-code --function-name testcdk \
+  --zip-file fileb://./target/lambda/server/bootstrap.zip
```
Comment (bigfoodK) [inline luda-editor/server/update-lambda.sh:2]: Is `fileb://` typo?

Comment (bigfoodK) [reply luda-editor/server/update-lambda.sh:2]: Ah, nevermind

**Review by bigfoodK — APPROVED** [review]

**Pull request merged**